### ISSUE TITLE
Make this easy to use on browsers and node from a simple import

### ISSUE THREAD

**Issue by mrchrisadams** [thegreenwebfoundation/co2.js#71]
Right now,  have a library that's a bit fiddly to use for both node and browsers, because we assume one runtime (i.e. nodejs) in places where we can't always make that assumption.

### the problem

We've had reports of imports with newer versions of webpack returning this error:

<img width="816" alt="Screenshot 2022-04-08 at 08 10 17" src="https://user-images.githubusercontent.com/17906/162605114-bfd78ed2-a04b-4de4-96cc-00a96d218504.png">

I think this is down to differences between nodejs runtime and browser runtimes.

There are definitely ways to feature detect for the platform you're on, (i.e. checking for the existence of node specific global variables and so on), and then include code accordingly, and in other repos we manage we've used tools like rollup.js to make browser specific builds.

### Acceptance criteria

This is issue is done when: 

1. there is a simple, browser friendly, 'compiled' version that people can include on a page with a script tag for making calculations
2. we have an example of using rollup to create distributed versions that work for browsers and for nodejs
3. there is a consistent, and easy to read documentation on how use both on the front page readme

**Comment by mrchrisadams** [comment]
I'm linking to PR #72 to show some ongoing work.

**Comment by mrchrisadams** [comment]
Hi @drydenwilliams  - what would the smallest test case be to replicate this error?

i.e. something like

1. start a project
2. install webpack CLI on computer
3. make entrypoint file importing/requiring CO2.js a dependency
4. run webpack with this entrypoint file as the input?

I've used webpack on previous projects, but I've tended to lean towards rollup.js for managing a build pipeline, as it's felt smaller and easier for me to understand what was going on, so I'm not sure how to reproduce this error to fix it.
